src/main/java/org/takes/rs/RsWithoutHeader.java

```diff
@@ -50,13 +51,14 @@ public RsWithoutHeader(final Response res, final CharSequence name) {
             new ResponseOf(
                 () -> {
                     final String prefix = String.format(
-                        "%s:", new EnglishLowerCase(name.toString()).string()
+                        "%s:", new UncheckedText(
+                            new Lowered(name.toString())
```

Comment:
@fanifieiev Isn't `name` already `String`? Why use `toString()` here?

Reply:
@fabriciofx Because name is CharSequence